Installation.md

```diff
@@ -5,6 +5,12 @@ There are a few ways to install Chocolatey. Chocolatey exists as a [nuget packag
 
 If you have Visual Studio 2010 and the NuGet extension installed, perhaps the quickest method is to use NuGet Package Manager. Three commands in succession and you are done. See below.
 
+## Windows 10
+If you have Windows 10 installed, you can add Chocolatey as a package source to the built-in, compatible package manager [OneGet](https://github.com/OneGet/oneget) by simply opening a PowerShell (as administrator) and issuing the following command:
+```powershell
+Register-PackageSource -Name chocolatey -Location http://chocolatey.org/api/v2 -Provider PSModule -Trusted -Verbose
+```
+

```

Comment:
Thanks! This appears incorrect though. Provider would be Chocolatey, not PSModule. PSModule isn't going to have the same functionality. Keep in mind we are fixing up the Chocolatey provider to be the official one. Right now I would leave a notice instead that the `Chocolatey has a very early preview in the built-in, compatible package manager aggregator [OneGet](link), and the official one is coming soon. Be sure to watch [chocolatey-onget](https://github.com/chocolatey/chocolatey-oneget) for more details!`